src/generator/AutoRest.Go/TransformerGo.cs

```diff
@@ -136,7 +136,7 @@ private void TransformModelTypes(CodeModelGo cmg)
                     name = $"{name}Type";
                 }
 
-                if (CodeNamerGo.Instance.UserDefinedNames.Contains(name))
+                if (CodeNamerGo.UserDefinedNames.Contains(name))
```

Comment:
If there was a push to have thing that way, I'm happy to change it. Though, it should be noted that I was changing it because I specifically don't think virtual makes much sense. :p